app/models/email_address.rb

```diff
@@ -11,6 +13,7 @@ class EmailAddress < ApplicationRecord
   # rubocop:disable Rails/HasManyOrHasOneDependent
   has_one :suspended_email
   # rubocop:enable Rails/HasManyOrHasOneDependent
+  has_many :identities, class_name: 'ServiceProviderIdentity', dependent: nil
```

Comment:
If we're not specifying `dependent` behavior, I'd suggest omitting it.

```suggestion
  has_many :identities, class_name: 'ServiceProviderIdentity'
```

Reply:
Got that. Rubocop was complaining. I enclosed it in the previous disable chunk.

Reply:
Yeah, I'm guessing it wants some sort of handling when the record is destroyed, which is what we're doing with the callback hook.